### PR TITLE
[plugin] add stashNotes

### PR DESCRIPTION
**Pull request by QxxxGit**

### CODE REVIEW

**Comment by scruffynerf** [comment]
Not working for me, no icon at all in corner when expected, didn't matter what browser, no other plugins/etc.  neither on latest Devel or v23.1

**Review by scruffynerf — COMMENTED** [review]
Works for me, however it needs the UI work by @WithoutPants in https://github.com/WithoutPants/stash/tree/ui-plugins-4

so on hold from approval until that code merged.

File: plugins/stashNotes/stashNotes.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            }
+        }, []);
+        return (React.createElement(React.Fragment, null,
+            React.createElement(NavButton, { onClickHandler: enableModal }),
+            React.createElement(NotesModal, { displayState: display, onCloseHandler: disableModal, onSaveHandler: saveNotes, notesState: notes, notesChangeHandler: (n) => setNotes(n) })));
+    };
+    const NavButton = ({ onClickHandler }) => {
+        const { Icon } = api.components;
+        return (React.createElement(React.Fragment, null,
+            React.createElement(Button, { className: "minimal d-flex align-items-center h-100", title: "Notes", onClick: onClickHandler },
```
Comment (DogmaDragon) [inline plugins/stashNotes/stashNotes.js:30]: Right now the button is misplaced compared to other buttons. Changing the class to `nav-utility minimal btn btn-primary` fixes the alignment.

**Comment by DogmaDragon** [comment]
I'm having the issue of the button not always loading. Sometimes I need to refresh the page several times for it to show. I assume it has to do with the order stuff is loaded, but that needs to be looked at.

**Comment by Tetrax-10** [comment]
We appreciate your contribution. Please note that CommunityScripts now adheres to strict formatting guidelines. Kindly refer to the [Contributing guide](https://github.com/stashapp/CommunityScripts?tab=readme-ov-file#contributing) for instructions.

**Review by Tetrax-10 — APPROVED** [review]

**Comment by Tetrax-10** [comment]
Looks good 👍.

**Pull request merged**